Practice/03_07/script.js

```diff
@@ -7,3 +7,21 @@
  * - Find an object that has another object inside of it to create a nested object.
  * - Test your objects in the browser console by accessing the entire object and its specific properties.
  */
```

Comment:
const house = {
    year: "1973",
    make: "Duplex",
    color: "Cream",
    model: {
        Side1: "two bedroom two bath",
        Side2: "one bedroom one bath"
    },
    houseOpen: false,
    newModel: function(Right, Left) {
        this.model.Side1 = Right;
        this.model.Side2 = Left;
    }
};

// Accessing the entire object and specific properties
console.log(house);
console.log("Year:", house.year);
console.log("Make:", house.make);
console.log("Color:", house.color);
console.log("Model (before):", house.model);
console.log("Model Side1:", house.model.Side1);
console.log("Model Side2:", house.model.Side2);

// Using the newModel method to update the model
house.newModel("three bedroom three bath", "two bedroom one bath");

// Checking the updates
console.log("Model (after):", house.model);